www/templates/manage.php

```diff
@@ -162,7 +162,7 @@ class="dcf-w-100% go_responsive_table flush-left dcf-table dcf-txt-sm"
                         </td>
                         <td data-header="Long URL">
                             <a
-                                href="<?php echo htmlspecialchars($lilurl->escapeURL($url->longURL)); ?>"
+                                href="<?php echo $lilurl->escapeURL($url->longURL) ?>"
```

Comment:
Missing semicolon, might not matter but it makes me feel better

Reply:
I have made the change